content/articles/exceptions-and-error/index.md

```diff
@@ -0,0 +1,268 @@
+# Exceptions and Error Handling in Python
+
+### Introduction
+
+Over the years, Python has gained acceptance in the programming world because of its simplicity and application across diverse areas, ranging from game development to web development, data science, and cyber security.
+
+Though Python is simple to work with, it is not that gracious to errors as whenever an error occurs during a program execution that makes Python unsure of what to do, and the program will halt and show a “traceback” error message which includes a report about the exception raised.
+
+Exceptions are special objects that manage errors raised during program execution; they can be handled with “try-except blocks,” which tells Python what to do when such an error occurs.
+
+This article will look at some significant errors in Python and how to handle them using the “try-except block”.
+
+###  Key Takeaways
+At the end of this article, the reader should have understood exceptions and how to handle them in Python.
+
+### Prerequisites
+To better understand this article, the reader is expected to have a basic understanding of Python and is familiar with working with any code editor. In this article,  the Jupyter notebook was used. 
+
+### Table of contents
+- [Keyboard interrupt error](#keyboard-interrupt-error)
+- [Python exception](#python-exception)
+- [Handling exceptions](#handling-exceptions)
+- [Handling multiple exceptions in one block of code](#handling-multiple-exceptions-in-one-block-of-code)
+- [The “Finally Block”](#the-“finally-block”)
+- [Conclusion](#conclusion)
+- [Further reading](#further-reading)
+
+Python 3 has 63 defined built-in exceptions, with all of them arranged to form a “tree-type” of hierarchy. Some of these exceptions are more general than others, as they include other exceptions, while others are entirely concrete as they represent only themselves. 
+
+![exception_hierarchy](exceptions-and-error/exception_hierarchy.png)
```

Comment:
Look into the docs to find the format for adding images.

Reply:
![Screen](https://user-images.githubusercontent.com/76263496/146716002-88e731f9-fad9-4e8d-97c7-e501c59f837f.jpg)

This was the format I saw, but I corrected it to what I have been using.